packages/google_maps_flutter/google_maps_flutter/test/fake_maps_controllers.dart

```diff
@@ -469,13 +429,13 @@ class FakePlatformGoogleMap {
 }
 
 class FakePlatformViewsController {
-  FakePlatformGoogleMap lastCreatedView;
+  FakePlatformGoogleMap? lastCreatedView;
```

Comment:
Can this be made `late`? It'd save a ton of `!`s in the test code.

Reply:
Alas no; see `reset()`. You can't assign a `late` variable back to null (even if you pinky-swear you'll set it to something again before using it).

Reply:
Oucccchhhh 🤕